admin/acceptance.py

```diff
@@ -585,8 +623,8 @@ def start_cluster(self, reactor):
         print "Assigning random tag:", random_tag
 
         for index in range(self.num_nodes):
-            name = "acceptance-test-%s-%s-%d" % (
-                self.creator, random_tag, index,
+            name = "%s-%s-%s-%d" % (
+                self.purpose, self.creator, random_tag, index,
```

Comment:
With the addition of the unique identifier, I would rather see the more distinguishing info at the start.  Move the purpose to the final part of the label (e.g. `jongiddy-lk3hEs-1-acceptance-testing`).